library/cartridge_check_instance_started.py

```diff
@@ -7,15 +7,18 @@
 
 argument_spec = {
     'control_sock': {'required': True, 'type': 'str'},
+    'stateboard': {'required': True, 'type': 'bool'},
 }
 
 
 def check_instance_started(params):
     try:
         control_console = get_control_console(params['control_sock'])
-        ok = control_console.eval('''
-            return require('membership').myself().status == 'alive'
-        ''')
+        ok = True
+        if not params['stateboard']:
+            ok = control_console.eval('''
+                return require('membership').myself().status == 'alive'
```

Comment:
Hmmm

I'm not sure that you could found in myself status anything except 'alive'... May be 'left'... But not dead I assume

Reply:
I think it's possible when invalid advertise URI is specified.